packages/backend-app-api/src/services/implementations/auth/authServiceFactory.test.ts

```diff
@@ -127,4 +138,90 @@ describe('authServiceFactory', () => {
       }),
     );
   });
+
+  it('should issue limited user tokens', async () => {
+    server.use(
+      rest.get(
+        'http://localhost:7007/api/auth/.well-known/jwks.json',
+        (_req, res, ctx) =>
+          res(
+            ctx.json({
+              keys: [
+                {
+                  kty: 'EC',
```

Comment:
bonus points for writing in a short comment how this key and the token signature below were generated, so that the test can be easily rewritten in the future if we update the set of claims or so?

Reply:
We refactored this into generating signatures on the fly instead